COMMITTERS.adoc

```diff
@@ -0,0 +1,164 @@
+= Information for Quarkus Committers
+
+This document contains useful information for Quarkus committers.
+
+[NOTE]
+====
+It is a work in progress, so if you think information is missing/unclear/inadequate,
+please reach out to us on the `quarkus-dev` mailing-list.
+====
+
+This document has a lot of information.
+It is not a set of strict rules, it is here to guide you.
+If you forget to do something, miss something, don't worry, it happens.
+
+Do your best and that will already be a lot :).
+
+== Merging Pull Requests
+
+The first thing is that, while being a committer,
+you can't really commit directly to the main repository:
+you need at least one review by your peers.
+
+Thus the need for a pull request.
+
+So you are more like a merger than a committer
+and you might merge more PRs from others than your own work.
+
+While not absolute, here are some advices:
+
+* if you really want a review by a specific person,
+  make it a named reviewer via a GitHub UI
+* if someone asked for a specific reviewer, better
+  give some time for this person to review the PR.
+  If not reviewed in a reasonable amount of time,
+  better ping him again to have some information
+  or ask the original author if we should go ahead.
+* if in doubt (on a specific part or because you're
+  not comfortable reviewing documentation or any other reason),
+  ask for help
+* except if it's a totally non brainer (typically a typo fix),
+  let the PR bake a few hours so some other people can have a
+  look if interested.
+  If the PR is very large or is a new extension, better let it
+  bake for a few days.
+* we do merge commits so please ask the author to properly
+  squash the commits before merging.
+  The idea is to have proper semantic commits.
+  If the author is not familiar with Git or not available,
+  you can also (carefully) do it for him as soon as you agreed on it.
+* make sure the commit comments are meaningful
+* at the beginning, you can start by giving your approval and not merging
+  and see how things evolve.
+  This should give you some hints of important things to check in the future.
+* only merge pull requests that target master.
+  Dealing with old branches is a separate process.
+* if you are sure a PR should go in and is just waiting for CI,
+  just use the `triage/waiting-for-ci` label.
+  It is a good hint for other committers.
+
+Obviously, each situation is different so use your own judgement,
+and if in doubt, just ask for advice, the other committers are here to help.
+
+== Pull Request Title
+
+The title should be in English, should not contain an issue number.
+It should also not contain ellipsis.
+
+If your commit message was too long and GitHub automatically cut
+the title, that's helpful if you can take the time to move the cut
+part where it belongs to have a full title.
+
+Titles are included in the Release notes so they are important.
+
+A good title would look like: `Fix this complex issue`.
```

Comment:
```suggestion
A good title would look like: `Introduce Hibernate Reactive extension`.
```

Reply:
Heh @machi1990 was faster than me. Thank you! :wink:

Reply:
I added your example too.

Reply:
Thank you!